solution/GraphicalDebugging/Util.cs

```diff
@@ -143,6 +143,15 @@ public static string CppRemoveCVRef(string type)
                     type = type.Remove(i - 1);
                 }
             }
+            if (type.Contains("!"))
+            {
+                //remove bracket-less module declaration sometimes appearing in the variable type (ends with "!" symbol)
+                int i = type.LastIndexOf("!");
+                if (i > 0) // including space
```

Comment:
What does it mean "include space"? Are there whitespaces or is this some different kind of space?
Is it possible for the type to be `!Type` e.g. because of some yet another issue in the debugger?
My point is, would it make sense to remove everything before and including `!` even if it was at the beginning, just to be safe?

Reply:
To be honest this is just a copy-paste effect from the previous condition checked in this function, so I actually add no comments of my own! I think that in general everything before and including the exclamation mark should be trimmed away. So far it worked fine in my environment which is rich of dependencies (so a good benchmark), but I don't know if this rule holds in general